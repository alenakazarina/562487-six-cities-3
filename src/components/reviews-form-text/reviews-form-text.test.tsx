import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ReviewsFormText from './reviews-form-text';

const mockFn = jest.fn();

describe(`ReviewsFormText`, () => {
  it(`should render ReviewsFormText correctly`, () => {
    const tree = renderer.create(
        <ReviewsFormText
          text={`Some text`}
          onChange={mockFn}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
