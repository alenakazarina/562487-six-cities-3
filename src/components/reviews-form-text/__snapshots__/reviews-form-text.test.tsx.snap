// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsFormText should render ReviewsFormText correctly 1`] = `
<textarea
  className="reviews__textarea form__textarea"
  id="review"
  name="review"
  onChange={[MockFunction]}
  placeholder="Tell how was your stay, what you like and what can be improved"
  value="Some text"
/>
`;
